### ISSUE TITLE
Lock with password

### ISSUE THREAD

**Issue by baudtack** [koreader/koreader#1606]
I'm not sure if this is in the realm of things that koreader should handle, but I used to lock my kindle keyboard with a password. I now use a kobo glo hd and koreader and it would be nice to have that functionality again.

**Comment by KenMaltby** [comment]
Not sure what you want to "lock" in koreader?  Do you mean a way to require a password to start koreader?  Or on return from suspend?

**Comment by baudtack** [comment]
Both ideally. But mostly return from suspend.

**Comment by lenisko** [comment]
Well, it's good idea, against kids or mom.

**Comment by KenMaltby** [comment]
Perhaps, if someone does take this on, it could be an offshoot of adding authentication to OPDS?

**Comment by gerroon** [comment]
I am dreamign the day these devices can have ecnrypted drives with properly secured aplications. If I loose my device today, someone can access to hundreds of books etc at no cost no problem.

For instance I found a broken ereader once (screen was cracked), it had all the account stuff, some books some music etc. Naturally I did not do anything malicious but it is possible to do it because these devices are not secured what so ever and no device companies seem to give a crap about including the big players.
